include/bgfx/c99/bgfx.h

```diff
@@ -90,6 +90,7 @@ typedef enum bgfx_renderer_type
     BGFX_RENDERER_TYPE_OPENGLES,              /** ( 7) OpenGL ES 2.0+                 */
     BGFX_RENDERER_TYPE_OPENGL,                /** ( 8) OpenGL 2.1+                    */
     BGFX_RENDERER_TYPE_VULKAN,                /** ( 9) Vulkan                         */
+    BGFX_RENDERER_TYPE_WEBGPU,                /** (10) WebGPU                         */
```

Comment:
Need to change scripts/bgfx.idl for this... Also bump bgfx version.

Reply:
Hmmmm, I have an `c:/Dev/bgfx/scripts/genie.lua:61: module 'bgfx-codegen' not found` when I try this (I'll come ask you on gitter)

Reply:
Found it. Done